src/GitReportCL/GitReport.CLI.csproj

```diff
@@ -12,20 +12,21 @@
     <Compile Remove="RunGitReport.cs" />
   </ItemGroup>
 
-  <ItemGroup>
-    <None Remove="jsconfig.json" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.1" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.EnvironmentVariables" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.FileExtensions" Version="2.1.1" />
     <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.1" />
+    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
   </ItemGroup>
 
   <ItemGroup>
     <Reference Include="System.Configuration">
```

Comment:
You need remove those types of refernces, this will prevent from running on other machines
* System.Configuration
* System.Web